cmd/dep/status.go

```diff
@@ -257,7 +262,12 @@ func (cmd *statusCommand) Run(ctx *dep.Ctx, args []string) error {
 					"Run `dep ensure` to regenerate the inputs-digest.")
 			}
 		default:
-			ctx.Out.Println("Failed to get status. Rerun with `-v` flag to see details.")
+			// Print the error in verbose mode.
+			if ctx.Verbose {
+				ctx.Err.Println(err)
+			} else {
+				ctx.Out.Println("Failed to get status. Rerun with `-v` flag to see details.")
+			}
```

Comment:
I was only thinking "don't suggest `-v` when it's already present". Isn't the same `err` already returned and logged higher up somewhere?

Reply:
oops! you're right, I overlooked that 😅 So, we get rid of the `default` part and there's no need to move the lock check.